### PR TITLE
chore: #1916 Purge unused code from admin portal: login page

### PR DESCRIPTION
**Pull request by trankhacvy**
# Pull request checklist

## Does this close any currently open issues?

fixes: https://github.com/reapit/foundations/issues/1916

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #

## What is the new behavior?

-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/admin-portal/src/components/pages/login.tsx [hunk 1]
```diff
@@ -1,98 +1,31 @@
...
-    const redirectRoute = getDefaultRouteByLoginType({
-      loginType,
-      isDeveloperFirstTimeLoginComplete,
-      isClientFirstTimeLoginComplete,
-    })
-
-    redirectToLogin(window.reapit.config.cognitoClientId, redirectRoute, loginType)
-  }
+export const onLoginButtonClick = () => {
+  const redirectRoute = `${window.location.origin}${Routes.ADMIN_APPROVALS}`
```
Comment (tanphamhaiduong) [inline packages/admin-portal/src/components/pages/login.tsx:17]: I think no need `ADMIN_` in prefix of routes

File: packages/admin-portal/src/core/private-route-wrapper.tsx [hunk 2]
```diff
@@ -30,108 +16,43 @@ export type PrivateRouteWrapperProps = {
...
-    /**
-     * loginType default in reducer = DEVELOPER
-     * when redirecting back to app after login
-     *   -> use state in url param (this was setted by cognito)
-     *   -> save the valid type to logintype
-     * further works will use loginType in reducer if param state isn't specificied in url param
-     */
-    loginType: type || loginType,
-  })
+  const route = `${window.location.origin}${Routes.ADMIN_APPROVALS}`
```
Comment (tanphamhaiduong) [inline packages/admin-portal/src/core/private-route-wrapper.tsx:32]: I think no need `ADMIN_` in the prefix

File: packages/admin-portal/src/core/router.tsx [hunk 3]
```diff
@@ -9,184 +9,26 @@ import { Info } from '@reapit/elements'
...
-                <PrivateRoute
-                  allow="DEVELOPER"
-                  path={Routes.DEVELOPER_HELP}
-                  exact
-                  fetcher
-                  component={DeveloperHelpPage}
-                />
-                <PrivateRoute allow="DEVELOPER" path={Routes.DEVELOPER_APP_PREVIEW} exact component={ClientAppDetail} />
-
+                <PrivateRoute allow="ADMIN" path={Routes.ADMIN_BILLING} component={DeveloperAdminBillingPage} exact />
```
Comment (tanphamhaiduong) [inline packages/admin-portal/src/core/router.tsx:31]: same as above

File: packages/admin-portal/src/core/router.tsx [hunk 4]
```diff
@@ -202,11 +44,11 @@ const Router = () => {
                   exact
                   fetcher
                 />
-                <PrivateRoute allow="ADMIN" path={Routes.ADMIN_STATS} component={AdminStats} exact fetcher />
+                <PrivateRoute allow="ADMIN" path={Routes.ADMIN_STATS} component={AdminStats} exact />
```
Comment (tanphamhaiduong) [inline packages/admin-portal/src/core/router.tsx:47]: same as above

File: packages/admin-portal/src/components/pages/login.tsx (hunk 1 above)
Comment (trankhacvy) [reply packages/admin-portal/src/components/pages/login.tsx:17]: @tanphamhaiduong all route name variables will be update in the next pull request. This pr just focused on login page.

**Pull request merged**